bulletroboy/cage_simulation.py

```diff
@@ -1,13 +1,17 @@
 import argparse
-import os
+import os, sys
 import pybullet as p
 import pybullet_data
-
 import rclpy
 from threading import Thread
 
+sys.path.append("..")
```

Comment:
I don't think that's necessary if you run it as a node

Reply:
deleted this